### DISCUSSION TITLE
Noobfragen Incoming (Homematic mit Intel NUC + Podman + RedHat 9)

### DISCUSSION THREAD

**Discussion by Mordecaine** [OpenCCU/OpenCCU#2942] [🇩🇪 Fragen&Antworten] [answered]
Hallo zusammen,

ich habe mir vor kurzem eine Heizungssteuerung von Homematic angeschafft. Ziel war es, das ganze in meine bestehende HomeAssistant (Podman Installation) einzubinden.

Ich habe mir daher einen HmIP-RFUSB Stick geholt.

Ich habe nun erst einmal versucht zu verstehen, wie diese ganze Zusammenarbeit funktioniert und es ungefähr so verstanden:
Home Assistant kommuniziert mit dem RaspberryMatic welches über den USB Stick mit meiner Heizungssteuerung von Homematic quatsch (wenn das hier schon nicht stimmt, interveniert bitte!)

Nun habe ich das RaspberryMatic via Podman an den start zu bekommen. Das hat eigentlich sehr gut funktioniert.

Der Dienst läuft, erkennt allerdings den USB Stick nicht. Den Teil mit den Kernel Modulen und dem pivccu habe ich übersprungen, da dieses ja nur auf der arm Architektur läuft.
Ich bin da einfach so naiv gewesen und dachte mir, dass es einfach auf Anhieb läuft.
Mein Problem ist nun, dass der Stick nicht erkannt wird:
```
Starting network: eth0: link up, fixed, firewall, inet up, 192.168.178.4, OK
Identifying Homematic RF-Hardware: HmRF: n/a, HmIP: n/a, OK
Updating Homematic RF-Hardware: HMIP-RFUSB: =>4.4.18, ERROR ( != 4.4.18)
```

Was absolut logisch wäre, wenn der Treiber nicht installiert ist.

Daher habe ich aktuell drei Fragen:
1. Macht das ganze mit Intel NUC + Podman + RedHat 9 alles einen Sinn?
2. Gibt es das Kernelmodul für den USB Stick auch in x86_64?
3. Sollte ich nicht lieber ein Proxmox installieren und mir eine Home Assistant VM erstellen?

Liebe Grüße

**Comment by jens-maus** [comment, marked as answer]
> 3\. Sollte ich nicht lieber ein Proxmox installieren und mir eine Home Assistant VM erstellen?

Genau das ist das was als "Best Practice" gilt und auf längere Sicht hin am besten wartbar ist und bleibt. Wenn es also die möglichkeit gibt ein proxmox einzusetzen, dann setze das so um statt mit Docker/Podman zu versuchen das lauffähig hinzubekommen.

**Reply by Mordecaine** [reply]
Danke dir für die schnelle Antwort. Dann werde ich das mal so umsetzen. Vielen Dank!

**Reply by Mordecaine** [reply]
Ah. Kurze Randfrage. Du würdest dann auch die Installation von RaspberryMatic über den Home Assistant Store empfehlen?

**Reply by jens-maus** [reply]
Nein, Wenn du schon Proxmox nutzt, dann HomeAssistant OS in ne extra VM und RaspberryMatic auch in ne extra VM. Und dann installierst du dir "nur" unter HomeAssistant die "Homematic (IP) Local" Integration und lässt sich gegen die RaspberryMatic VM binden. Das RaspberryMatic HA Addon nutzt man nur wenn es keine Möglichkeit gibt eine Vollvirtualisierung wie Proxmox nutzen zu können.
